conans/client/output.py

```diff
@@ -73,24 +73,32 @@ def is_terminal(self):
     def writeln(self, data, front=None, back=None, error=False):
         self.write(data, front, back, newline=True, error=error)
 
+    def _write(self, data, newline=False):
```

Comment:
I fail to see the advantage of having these functions

Reply:
It was trying to avoid to redefine the entire ConanOutput write here:

https://github.com/conan-io/conan/blob/45f4b58c9063dac6670e60082600ee00767b27fe/conans/util/progress_bar.py#L13-L23

Maybe I can simplify it to just one function?

Reply:
Or simply avoid them completely. What are they improving? I think it does exactly the same, but more code, why not leave it as-is.